calc_bmi_read_write_another_file/main.py

```diff
@@ -0,0 +1,206 @@
+import os
+
+def print_menu():
+  file = open('calc_bmi_read_write_another_file/menu.txt', encoding='utf-8')
```

Comment:
utf-8 is set by default

Reply:
![image](https://user-images.githubusercontent.com/129417446/232521470-fbe246a0-1fab-4e1d-a4ae-aec7ce996c7a.png)